packages/core/src/codewhisperer/util/customizationUtil.ts

```diff
@@ -116,25 +112,27 @@ export const getSelectedCustomization = (): Customization => {
     if (selectedCustomization && selectedCustomization.name !== '') {
         return selectedCustomization
     } else {
-        const customizationFeature = FeatureConfigProvider.getFeature(Features.customizationArnOverride)
-        const arnOverride = customizationFeature?.value.stringValue
-        const customizationOverrideName = customizationFeature?.variation
-        if (arnOverride === undefined) {
-            return baseCustomization
-        } else {
-            return {
-                arn: arnOverride,
-                name: customizationOverrideName,
-                description: baseCustomization.description,
-            }
-        }
+        return baseCustomization
     }
 }
 
-export const setSelectedCustomization = async (customization: Customization) => {
+/**
+ * @param customization customization to select
+ * @param isOverride if the API call is made from us (Q) but not users' intent, set isOverride to TRUE
```

Comment:
> API call is made from us (Q)

Do you mean from the extension side?

Reply:
yea, i want to express it's us to override users' selection and it's not on their own intention to change such configuration